### DISCUSSION TITLE
Tutorials

### DISCUSSION THREAD

**Discussion by alexanderson1993** [Thorium-Sim/thorium-nova#579] [Ideas]
Before reading this, stop and watch this video. Think about your experiences learning how to use other bridge simulators. 

https://www.youtube.com/watch?v=-GV814cWiAw

Thorium Nova won’t have training in the same sense as Thorium Classic, since there are so many variables in how the game is trained. 

- How many players are playing at that time?
- How many noob vs experienced players are there? Can the experienced players help the noob players?
- What kind of mission are they doing?
- Is there a flight director running the mission? 
- How complicated of a ship are they running? Do they have internal crew and cargo? Are there other things affecting how their station runs?

In the space centers where Thorium Classic is run, there is dedicated staff who can lower the pressure of training:

- They can answer questions in the middle of training if they don’t understand something.
- They can answer questions and re-train in the middle of the flight with a flight director.

This video shows a lot of interesting ideas, including

- Having an advisor character that’s part of the game that can tell you when you do something dumb or aren’t noticing something.
- Having a separate tutorial section outside of the game itself that lets players drill and practice how to do certain things on stations.
- Distributing training throughout the gameplay, so a player doesn’t have to learn everything upfront, but can slowly learn things as they need.
- Or building the tutorial into the game itself by easing players into doing different parts of their stations.

On that last point, at the CMSC, the founder, Vic, would always start his flights the same way. The crew was trained using honest-to-goodness tape cassettes before the mission began, so they had an idea of how to do their jobs. As Flight Director, Vic would be a character from Engineering calling up to the bridge and introduce himself to the captain. He would then go around the bridge and introduce every member of the crew to the captain, telling the captain what that person's job entailed. Sometimes it would coincide the things the Flight Director was doing. For example, he would say "Let's turn our attention to Nolan, who's in charge of the warp engines. Now that we're on course, we can activate our engines. So, Captain, say 'Nolan'" and he would wait for the captain to respond, and then continue "And say 'Warp' and pick a speed to go, from 1 to 9" and then the captain would give the order.

Most likely the captain already knew his crew - they came to the space center together. But this helps the captain know what capabilities their ship has, who was in charge of what, and as a bonus the crew members would get a quick refresher on how to do their stations.

It would be great to integrate stuff like that into the game somehow. 

There will have to be some kind of multi-modal situation. Thorium Classic has text-based training that you can open up. It guides you step-by-step through how all of the controls work, but it's a little clunky.

Video or audio training could be valuable, but also takes up a lot of time. It'd be good to have the training experience be more seamless and integrated.

We also need to consider tutorials for the Flight Director controls, since that will be just as important to the gameplay as the crew stations, though like a DM, we can assume the Flight Director will take time to familiarize themselves with their station before the game starts, so their training doesn't have to be quite so integrated in with the gameplay.

A few other options we could consider (more or less in order of how much I want to implement this kind of thing):
- Tutorial missions, which are fully automated and hand-hold each crew member through different parts of their controls as they complete a very simple mission. 
- Have single-player missions where the player plays a single station that's part of a larger bridge, but the rest of the bridge is fully automated so that player only has to focus on the single station.
  - These could progress through different techniques. Beginner teaches you the bare minimum to get by, intermediate shows a couple of interesting tricks, advanced demonstrates how a power-user would run the station.
- Changing the on-screen text training into a help menu, which highlights the controls with little bubbles that you can click to remind yourself how the thing works, instead of doing the step-by-step thing.
- Video trainings on YouTube, so folks could watch the training beforehand to get an idea for how the station works.
- Written documentation explaining how things work, and maybe part of it is searchable from the crew station so they can quickly reference something.

I'd love to hear everyone else's thoughts.

**Comment by GlynD02** [comment]
Hi Alex I like the little tutorials of each station in Empty Epsilon, ideal for brand new users.

Played the fore-runner of DCS world ages ago and those had a set of tutorials that ran through sections of A-10s operation in nice easy chunks with on-screen highlighting of the system being trained - the YT playlist is here if you are interested https://www.youtube.com/watch?v=dk5hN3u6FmA&list=PLFFC261F5C6F728DA.

Once you have got more stations running I am sure the community will pitch in and help out.

Cheers
Glyn
